tests/frontend/test_kibana_app/cypress/integration/utils/constants.js

```diff
@@ -0,0 +1,7 @@
+export const BASIC = 'basic';
+export const ODFE = 'odfe';
+export const ODFE_PASSWORD = 'admin';
+export const ODFE_USERNAME = 'admin';
+export const XPACK = 'xpack';
+export const XPACK_PASSWORD = 'elastic';
+export const XPACK_USERNAME = 'elastic';
```

Comment:
`User` and `password` should be obtained from .env files

Reply:
This is a good idea. But we will continue to use the current approach because nowadays our environments are generated from a docker-compose file that has defined these constants already defined. 
A better approach would be to read those files and get the strings from there.
This will be useful when moving to a CI env.